src/actors/rqgActorSheet.ts

```diff
@@ -694,6 +695,33 @@ export class RqgActorSheet extends ActorSheet<
       });
     });
 
+    this.form?.querySelectorAll("[data-reputation-roll]").forEach((el) => {
+      let clickCount = 0;
+      el.addEventListener("click", async (ev: Event) => {
+        clickCount = Math.max(clickCount, (ev as MouseEvent).detail);
+
+        if (clickCount >= 2) {
+          // @ts-ignore wait for foundry-vtt-types issue #1165 #1166
+          const speakerName = this.token?.name || this.actor.data.token.name;
+          await CharacteristicCard.roll(
+            "Reputation",
+            this.actor.data.data.background.reputation || 0,
+            0,
+            0,
+            this.actor,
+            speakerName
+          );
+          clickCount = 0;
+        } else if (clickCount === 1) {
+          await ReputationCard.show(
+            this.actor,
+            // @ts-ignore wait for foundry-vtt-types issue #1165 #1166
+            this.token
+          );
```

Comment:
There needs to be a setTimeout around this, otherwise it will always show a chatcard as well. Have a look at the other listeners.

Reply:
Done